jsonschema/_format.py

```diff
@@ -158,7 +158,7 @@ def is_host_name(instance):
 
 
 try:
-    import rfc3987
+    import rfc3986
```

Comment:
Great, I can manage the code change - but I'll probably need guidance around tox since I'm not familiar with it